Anlab.Mvc/Client/components/Summary.tsx

```diff
@@ -22,56 +19,32 @@ interface ISummaryProps {
 
 export class Summary extends React.Component<ISummaryProps, any> {
     totalCost = () => {
-        const grindTotal = this.grindCost() * this.props.quantity;
-        const foreignSoilTotal = this.foreignSoilCost() * this.props.quantity;
-        const filterWaterTotal = this.waterFilterCost() * this.props.quantity;
 
         const total = this.props.testItems.reduce((prev, item) => {
             // total for current item
             const price = this.props.payment.clientType === 'uc' ? item.internalCost : item.externalCost;
             const perTest = price * this.props.quantity;
 
-            return prev + perTest + item.setupCost;
+            return prev + perTest + (this.props.payment.clientType === 'uc' ? item.internalSetupCost : item.externalSetupCost);
```

Comment:
@jSylvestre setup cost is not the same for internal/external so we'll still need to figure this out